language/move-prover/bytecode-to-boogie/test_mvir/verify-stdlib/libra_account.prover.bpl

```diff
@@ -12,7 +12,3 @@ procedure {:inline 1} LibraAccount_save_account (arg0: Value, arg1: Value) retur
 procedure {:inline 1} LibraAccount_write_to_event_store (tv0: TypeValue, arg0: Value, arg1: Value, arg2: Value) returns ();
   requires ExistsTxnSenderAccount(m, txn);
   ensures old(b#Boolean(Boolean(true))) ==> !abort_flag;
-
-function {:inline 1} max_balance(): Value {
```

Comment:
I removed max_balance() here and added max_u64() to prelude.bpl because I found myself use this value all the time in both coin and account.